packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/impactanalysis/services/ImpactAnalysisMessageService.java

```diff
@@ -0,0 +1,70 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.application.impactanalysis.services;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import org.eclipse.emf.ecore.EObject;
+import org.eclipse.emf.ecore.EStructuralFeature.Setting;
+import org.eclipse.sirius.components.core.api.ILabelService;
+import org.eclipse.sirius.web.application.impactanalysis.services.api.IImpactAnalysisMessageService;
+import org.springframework.stereotype.Service;
+
+/**
+ * Provides impact analysis messages.
+ *
+ * @author gdaniel
+ */
+@Service
+public class ImpactAnalysisMessageService implements IImpactAnalysisMessageService {
```

Comment:
This was defined in `InvokeUpdateLibraryImpactAnalysisHandler`, but I needed it also in the code that runs the impact analysis for the removal of a dependency, so I factorized it in a reusable service.